mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -217,7 +217,10 @@ def do_cor_fit(self):
         self.model.do_fit()
         self.view.set_results(*self.model.get_results())
         self.do_update_projection()
-        self.do_preview_reconstruct_slice()
+        try:
+            self.do_preview_reconstruct_slice()
+        except Exception:
+            pass
```

Comment:
Can this be a specific exception? Not quite clear what we're trying to capture

Reply:
Astra literally raises type Exception, here so unfortunately not.
```  File "/home/sam/miniconda3/envs/mantidimaging/lib/python3.7/site-packages/astra/projector.py", line 36, in create
    return p.create(config)
  File "astra/projector_c.pyx", line 67, in astra.projector_c.create
Exception: Unable to initialize Projector2D.```

Reply:
I will add a comment to explain this

Reply:
That's annoying but not much we can do